Mathlib/Analysis/SpecialFunctions/ImproperIntegrals.lean

```diff
@@ -177,3 +177,36 @@ theorem integral_Ioi_cpow_of_lt {a : ℂ} (ha : a.re < -1) {c : ℝ} (hc : 0 < c
   simp_rw [neg_neg, Complex.norm_eq_abs, Complex.abs_cpow_eq_rpow_re_of_pos hx, Complex.add_re,
     Complex.one_re]
 #align integral_Ioi_cpow_of_lt integral_Ioi_cpow_of_lt
+
+theorem intervalIntegral_inv_one_add_sq_tendsto :
+    Tendsto (fun i => ∫ (x : ℝ) in -i..i, (1 + x ^ 2)⁻¹) atTop (𝓝[<] π) := by
+  convert TendstoNhdsWithinIio.const_mul (b := 2) (by norm_num) arctan_atTop <;> simp [two_mul]
+
+@[simp]
+theorem integrable_inv_one_add_sq : Integrable fun (x : ℝ) ↦ (1 + x ^ 2)⁻¹ := by
```

Comment:
this is a special case of [integrable_rpow_neg_one_add_norm_sq](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/SpecialFunctions/JapaneseBracket.html#integrable_rpow_neg_one_add_norm_sq)